pcsx2/GS/Renderers/Common/GSDevice.h

```diff
@@ -749,9 +749,8 @@ class GSDevice : public GSAlignedClass<32>
 protected:
 	static constexpr int NUM_INTERLACE_SHADERS = 5;
 	static constexpr float MAD_SENSITIVITY = 0.08f;
-	static constexpr u32 MAX_POOLED_TARGETS = 300;
+	static constexpr u32 MAX_POOLED_TARGETS_OR_TEXTURES = 300;
```

Comment:
This removes the flexability of having them separate if we need larger pools for one or the other in the future.

Reply:
We can just split them then, otherwise we have unnecessary checks now.

Reply:
To be honest I'd prefer if it was left as-is, I specifically PR'ed it split so it can be tweaked easily when profiling.

Reply:
Indeed, for profiling it's useful, will hide it in the analyser, maybe some others as well.